UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -37,36 +41,11 @@ public override bool IsPassable(Vector3Int from)
 		return IsPassable();
 	}
 
-	protected override void OnRotationStart(RotationOffset fromCurrent, bool isInitialRotation)
-	{
-		base.OnRotationStart(fromCurrent, isInitialRotation);
-		if (!isInitialRotation)
-		{
-			UpdateManager.Instance.Add(RemainUpright);
-		}
-	}
-
-	void RemainUpright()
```

Comment:
Looks like you're rolling back quite a few features. Did you discard all new code when resolving merge conflicts?

Reply:
my bad